src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9342,9 +9350,12 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
         }
     } else if (VRFCoolingMode || HRCoolingMode) {
         // IF the system is in cooling mode and/or the terminal unit requests cooling
-        if (NoCompOutput <= QZnReq) {
-            PartLoadRatio = 0.0;
-            return;
+        if (NoCompOutput <= QZnReq && ((QZnReq < -DataHVACGlobals::SmallLoad) || (QZnReq >= -DataHVACGlobals::SmallLoad && !HRCoolingMode))) {
```

Comment:
Should this be `...||(QZnReq >= DataHVACGlobals::SmallLoad ...` (no minus sign here)?

Reply:
Agree.

Reply:
Done.